SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/AsyncTests.swift

```diff
@@ -180,5 +180,9 @@ class AsyncTests: XCTestCase {
     func testSwiftCallsRustAsyncFnRetStruct() async throws {
         let _: AsyncRustFnReturnStruct = await rust_async_return_struct()
     }
+    
+    func testSwiftCallsRustAsyncFnReturnResultNullOpaqueRust() async throws {
+        try await rust_async_func_return_result_null_opaque_rust()
+    }
```

Comment:
Let's test both the ok and error case (I see that the other tests don't, but don't want to continue that habit).

One way would be to pass in a bool to `rust_async_func_return_result_null_opaque_rust` whether or not to return an error.

Then call it twice with `let _: () = try await  ...` and `let _: NameOfRustType = try await ...`

Can do this however you want though